adops/gam-hbt-step-by-step.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page_v2
+title: Header Bidding Trafficking with Prebid Step by Step
+head_title: Header Bidding Trafficking with Prebid Step by Step
+description: Step-by-step instructions for setting up Header Bidding Trafficking in GAM for Prebid.
+#note the sidebar type needs to reflect the section this file is displayed in. See _data/sidenav.yml for the side nav categories.
+sidebarType: 3
+---
+
+# Header Bidding Trafficking with Prebid Step by Step
+
+{: .no_toc }
+
+- TOC
+{: toc }
+
+Publishers new to header bidding can use this guide to set up header bidding trafficking for Prebid in Google Ad Manager. Unlike a line item setup, header bidding trafficking provides publishers with precise bid prices to compete in the auction and no need for shell creatives. It also provides unified reporting, and a straightforward workflow. These instructions describe only the specific settings required for header bidding trafficking; they are not intended to be comprehensive instructions that replace or duplicate the GAM documentation.
+
+- If you prefer to use line items to manage your header bidding setup, visit [Line item setup](/adops/step-by-step.html).
+- For how to migrate from line items to header bidding trafficking, visit [Header bidding trafficking](https://support.google.com/admanager/answer/12273163) and the corresponding [FAQ](https://support.google.com/admanager/answer/12270263).  These instructions also describe how to run an experiment to show potential revenue uplift from migrating to header bidding trafficking from a line item setup. 
```

Comment:
'alternatively, publishers could set a randomized  key value pair in targeting and setup a protection preventing HBT from cooccurring with the key value pair to validate metrics.'

Reply:
@patmmccann that's an interesting take. Could you provide this in a separate PR and we merge this? 

The one sentence is probably not enough information to unpack this. However it might be super useful to publishers migrating to see what alternative validations are possible.